sci-astronomy/stellarium/stellarium-25.3.recipe

```diff
@@ -5,7 +5,7 @@ binoculars or a telescope."
 HOMEPAGE="https://stellarium.org"
 COPYRIGHT="2000-2025 Stellarium Developers"
 LICENSE="GNU GPL v2"
-REVISION="2"
+REVISION="3"
```

Comment:
No need for a revision bump only for 32bit. For 64bit build has completed on buildmaster and is finishing up.

Reply:
But currently 64bit is building via GCC :-/

Reply:
Would this change force it to use clang++ ? (checking the log from buildmaster it already takes that package into the list at the beginning).